config/smarty.config.inc.php

```diff
@@ -229,3 +232,25 @@ function smarty_endWithoutReference($arrayValue)
 {
     return end($arrayValue);
 }
+
+function renderTemplate(array $params) {
+    $twigTemplate = $params['twig_template'];
+    unset($params['twig_template']);
+    $smartyTemplate = $params['smarty_template'];
+    unset($params['smarty_template']);
+
+    if (
+        SymfonyContainer::getInstance()
```

Comment:
It's up to you ;)

Reply:
I think that it could be usefull to merge my refacto first and to use FeatureFlagManager on this PR.
I write some tests and my PR can be mergeable as soon as possible!
What do you think?